src/styles/blocks/stars.css

```
@@ -0,0 +1,32 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../../images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star {
+  background-image: url(../../images/star.svg);
+}
+
+.stars--1 .stars__star:nth-child(n+2) {
+  background-image: url(../../images/star.svg);
+}
+
+.stars--2 .stars__star:nth-child(n+3) {
+  background-image: url(../../images/star.svg);
+}
+
+.stars--3 .stars__star:nth-child(n+4) {
+  background-image: url(../../images/star.svg);
+}
+
+.stars--4 .stars__star:nth-child(n+5) {
+  background-image: url(../../images/star.svg);
+}
```

```suggestion
.stars--1 :nth-child(-n+1),
.stars--2 :nth-child(-n+2),
.stars--3 :nth-child(-n+3),
.stars--4 :nth-child(-n+4),
.stars--5 :nth-child(-n+5) {
  background-image: url(../../images/star-active.svg);
}
```
suggest to make selectors more consistent with modifiers (same number in modifier and nth-child) and set by default star and in midifiers star-active.
it is possible to reduce amount of code by separate selector with same styles by comma.